server/test/converterRouter/converter.test.js

```diff
@@ -0,0 +1,41 @@
+const supertest = require('supertest');
+
+const app = require('../../app');
+const request = supertest(app);
+
+describe('POST /api/specification/invalid/openapi_3', () => {
+  it('should return 404 for invalid conversion', (done) => {
```

Comment:
not really invalid, it's just that we aren't supporting it. Could be reworded as for unsupported conversion

Reply:
Also instead of a number it's better to say Not Found/Bad Request or something in words because tests are useful documentation for non technical folks as well.